src/disco/FixedWidthField/Real/ENDF/test/write.test.cpp

```diff
@@ -8,27 +8,111 @@
 
 SCENARIO("FixedPoint write", "[FixedPoint], [write]"){
   {
+    constexpr double infinity = std::numeric_limits<double>::infinity();
+
     std::vector< std::pair< double, std::string > >
-      testSet = { { 0.0, " 0.000000+0" }, 
-                  { 10.0, " 1.000000+1" },
-                  { 3.14159, " 3.141590+0"},
-                  { std::numeric_limits<double>::infinity(), "   Infinity"},
-                  { 0.69314718056, " 6.931472-1"},
-                  { 1E-100, "***********" },
-                  { 1E10, " 1.00000+10" },
-                  { 1E-10, " 1.00000-10" },
-                  { -1E10, "-1.00000+10" },
-                  { -1E-10, "-1.00000-10" },
-                  { -10.0, "-1.000000+1" },
-                  { -3.14159, "-3.141590+0"},
-                  { -std::numeric_limits<double>::infinity(), "  -Infinity"},
-                  { -0.69314718056, "-6.931472-1"} };
+      testSet = { 
+        { 0.0,             " 0.000000+0" },  
+        { 10.0,            " 1.000000+1" },
+        { 3.14159,         " 3.141590+0"},   
```

Comment:
This is of course correct, but it hurts to see that 0 instead of the 6th digit of pi.